abcclassroom/clone.py

```diff
@@ -168,36 +168,31 @@ def copy_assignment_files(config, student, assignment_name):
     -----------
     config: dict
         config file returned as a dictionary from get_config()
-    student:
+    student: string
+        Name of the student who's files are being copied
     assignment_name: string
         Name of the assignment for which files are being copied
 
     """
     course_dir = cf.get_config_option(config, "course_directory", True)
     materials_dir = cf.get_config_option(config, "course_materials", False)
     clone_dir = cf.get_config_option(config, "clone_dir", True)
+    ignore_files = cf.get_config_option(config, "files_to_ignore", False)
     repo = "{}-{}".format(assignment_name, student)
 
     # Copy files from the cloned_dirs/assignment name directory
-    # TODO - right now this ONLY copies notebooks but we may want to copy
-    # other file types like .py files as well.
-    files = Path(course_dir, clone_dir, assignment_name, repo).glob("*.ipynb")
+
+    source_dir = Path(course_dir, clone_dir, assignment_name, repo)
     destination = Path(
         course_dir, materials_dir, "submitted", student, assignment_name
     )
     destination.mkdir(parents=True, exist_ok=True)
-    print(
-        "Copying files from {} to {}".format(
-            Path(clone_dir, repo), destination
-        )
+    print("Copying files from {} to {}".format(Path(source_dir), destination))
+
+    # Using the copytree function from util to make copying easier
+    shutil.copytree(
```

Comment:
ok i just played with this. it does work however because it is copying everything i had to add a bunch of things to files_to_ignore. some of the things in files_to_ignore i'd want to move in the template  move so maybe this one function handles moving files differently than the 3 other places??